spec.md

```diff
@@ -164,49 +205,105 @@ echo "SPDX-License-Identifier: MIT" > file.txt
 # REUSE-IgnoreEnd
 ```
 
-### In-line snippet comments
+### REUSE.toml
 
-If a copyright and/or licensing info is to apply only to a certain snippet
-instead of the whole file, SPDX snippet tags SHOULD be used (as defined in [SPDX
-Specification, Annex H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
+Copyright and Licensing Information MAY be associated with a file through a
+`REUSE.toml` file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
```

Comment:
Given the intentions of the REUSE project, I would use the opportunity to underline the value of per-file information:
```suggestion
impossible or undesirable. Note that Copyright and Licensing Information per
file is preferred because it is more explicit and more resistant when files
are moved or copied.
```

Reply:
Agreed, but this is the wrong place. I am writing this in the 'Comment headers' section instead.